<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>delete_s2</class>
 <widget class="QDialog" name="delete_s2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_userNo">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Check UserNo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>UserNo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ItemNo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_itemNo">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Check ItemNo</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textEdit_userNo">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textEdit_itemNo">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delete">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>410</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Review</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>221</y>
     <width>661</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_search">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_year">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_month">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>410</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_day">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>410</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>410</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
